### ISSUE TITLE
Blog: Post 1

### ISSUE THREAD

**Issue by bwklein** [thegooddocsproject/website-hugo#128]
Topic: TBD

**Comment by ryanmacklin** [comment]
Have a new branch with first post copy and my bio up:
https://github.com/thegooddocsproject/website-hugo/blob/ryanmacklin/blog-post-128/content/en/blog/first.md
https://github.com/thegooddocsproject/website-hugo/blob/ryanmacklin/blog-post-128/content/en/author/ryan_macklin.md

@flicstar Could you review and apply any other edits to those? I wasn't sure how to quickly address your open-ended note regarding the bullet section, but I trust you to revise —I'm not precious about that specific arrangement of info, and if you see another opportunity, revise away!

Open issues:

- This needs a stock photo head. I'm okay if this gets published with what we have right now, as long as we can swap out later. Having a legit publication date is paramount.
- I'm using code block for "example" copy, which doesn't feel like precisely the right formatting. But we can address this post-launch. Again, legit publication date is paramount.

Basically, when the content is ready, I'm okay with getting it published. In the future, we'll want to have a solid process for the stock issue side of things, and have posts drafted up at least a week before release. But this is very fly-by-the-seats-of-our-pants stage of release. (Provided others are alright with this approach, since it's a periodical and not a product.)

**Comment by ryanmacklin** [comment]
And Bryan, could you see if there are any execution issues with this? I'm not sure if the dates ended up working the way I expected.

**Comment by zivbk1** [comment]
@ryanmacklin can you make a PR for your branch into main?

**Comment by ryanmacklin** [comment]
Done and linked!
